.github/workflows/trigger.yml

```diff
@@ -0,0 +1,34 @@
+name: Tag & Release Binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: Version number of the release
+        default: 0.5.0
+        required: true
+
+jobs:
+  bump_version:
+    name: Version bump
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Tag a Release
+      uses: actions/create-release@latest
+      id: create_release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.event.inputs.releaseVersion }}
+        release_name: Release ${{ github.event.inputs.releaseVersion }}
+        body: "TODO: edit release notes before publishing"
```

Comment:
Might want to look at
https://github.com/marketplace/actions/release-notes-generator